06-setup-backup-restore/trilio.md

```diff
@@ -403,31 +417,27 @@ Now, please go ahead and check if the `Target` resource created earlier is `heal
 ```shell
 kubectl get target trilio-s3-target  -n tvk
 ```
-
 The output looks similar to (notice the `STATUS` column value - should be `Available`, meaning it's in a `healthy` state):
 
 ```text
 NAME               TYPE          THRESHOLD CAPACITY   VENDOR   STATUS      BROWSING ENABLED
 trilio-s3-target   ObjectStore   10Gi                 Other    Available
 ```
-
 If the output looks like above, then you configured the S3 target object successfully.
 
 **Hint:**
-
 In case the target object fails to become healthy, you can inspect the logs from the `trilio-s3-target-validator` Pod to find the issue:
 
 ```shell
 # First, you need to find the target validator
 kubectl get pods -n tvk | grep trilio-s3-target-validator
 
 # Output looks similar to:
-# trilio-s3-target-validator-tio99a-6lz4q              1/1     Running     0          104s
+trilio-s3-target-validator-tio99a-6lz4q              1/1     Running     0          104s
```

Comment:
Please do comment this line, as it was before. The reason is that if the user will copy the entire block of code (using the GitHub copy button), and paste it in the shell, it will fail.

Reply:
Sounds good Cristian! Commented line now. Thank you!